codesign/lib/src/file_codesign_visitor.dart

```diff
@@ -196,7 +243,7 @@ update these file paths accordingly.
       if (entity is io.Directory) {
         await visitDirectory(
           directory: directory.childDirectory(entity.basename),
-          entitlementParentPath: entitlementParentPath,
+          entitlementParentPath: joinEntitlementPaths(entitlementParentPath, entity.basename),
```

Comment:
When would `entitlementParentPath` not be the parent, but instead the entity itself?  This is confusing...